extensions/wikia/Follow/FollowHelper.class.php

```diff
@@ -220,7 +223,7 @@ static public function watchBlogListing(&$article, &$user, $text, $summary, $min
 				$title = Title::makeTitle( NS_BLOG_LISTING, $title );
 				$related[] = ucfirst($title->getDBKey());
 			}
-			self::emailNotification($article->getTitle(), $related, NS_BLOG_LISTING, $user, "blogpost", wfMsg('follow-bloglisting-summary'));
+			self::emailNotification($article->getTitle(), $related, NS_BLOG_LISTING, $user, self::LOG_ACTION_BLOG_POST, wfMsg('follow-bloglisting-summary'));
```

Comment:
Sure, and while I'm at it I'll UPDATE SPACING FOR THE WHOLE FILE!  And add some comments.  While I'm at it.